Examples/ObjectiveC/CacheManagementExample.m

```diff
@@ -0,0 +1,149 @@
+#import "CacheManagementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleCacheManagement = @"CacheManagementExample";
+
+@interface CacheManagementExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) UIButton *alertButton;
+@property (nonatomic) UIAlertController *alertController;
+@end
+
+@implementation CacheManagementExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.delegate = self;
+    [self.view addSubview:self.mapView];
+
+    // Create an offline pack.
+    [self addOfflinePack];
+
+    // Add a bar button. Tapping this button will present a menu of options. For this example, the cache is managed through the UI. It can also be managed by developers through remote notifications.
+    // For more information about managing remote notifications in your iOS app, see the Apple "UserNotifications" documentation: https://developer.apple.com/documentation/usernotifications?language=objc
+    UIBarButtonItem *alertButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemOrganize target:self action:@selector(presentActionSheet)];
+    [self.navigationController.navigationItem setRightBarButtonItem:alertButton];
```

Comment:
The navigation controller actually belongs to the parent view controller here, which is why this line doesn’t make the button appear on the nav bar.

Reply:
You may also be able to skip the `navigationController` part entirely by using `parent.navigationItem`.